src/libstd/sys/unix/mod.rs

```diff
@@ -73,13 +73,12 @@ pub fn init() {
     // errors are ignored while printing since there's nothing we can do about
     // them and we are about to exit anyways.
     fn oom_handler() -> ! {
-        use intrinsics;
         let msg = "fatal runtime error: out of memory\n";
         unsafe {
             libc::write(libc::STDERR_FILENO,
                         msg.as_ptr() as *const libc::c_void,
                         msg.len() as libc::size_t);
-            intrinsics::abort();
+            rtabort!();
```

Comment:
This was somewhat intentionally done because as part of the OOM handler we want to be sure that no allocations happen here. Right now `dumb_print` (called by `rtabort!`) uses `Stderr::new()` which does a bunch of allocations on Windows (for random unicode tidbits), which may not be appropriate to call in an OOM situation.